### PR TITLE
Create reporter function for presenting information to the user

### LINKED ISSUE #147: Create reporter function for presenting information to the user
The 'expect' function should present the contents of the FluentResult object in a clear way. It should also be configurable in the way that the user should be able to specify the information wanted. The function should work in a similar way to fast-check's assert.

### PR DESCRIPTION
**Pull request by tiagoacastro**
Closes #147

### CODE REVIEW

**Review by hugoferreira — CHANGES_REQUESTED** [review]
Can we please have a test demonstrating this capability?

**Review by tiagoacastro — COMMENTED** [review]
I have added the tests for the FluentReporter, let me know if you find them sufficient

File: test/reporter.test.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    fc.expect(fc.scenario()
+      .forall('a', fc.integer(-10,10))
+      .forall('b', fc.integer(-10,10))
+      .then(({a, b}) => a + b === (a - 1) + (b + 1))
+      .check()
+    )
+  })
+  it('Returns error on a unsatisfiable property', () => {
+    expect(
+      function(){
```
Comment (hugoferreira) [inline test/reporter.test.ts]: `() => {`

File: test/reporter.test.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    fc.expect(fc.scenario()
+      .forall('a', fc.integer(-10,10))
+      .forall('b', fc.integer(-10,10))
+      .then(({a, b}) => a + b === (a - 1) + (b + 1))
+      .check()
+    )
+  })
+  it('Returns error on an unsatisfiable property', () => {
+    expect(
+      function () {
```
Comment (arestivo) [inline test/reporter.test.ts:17]: User lambda functions

**Review by hugoferreira — CHANGES_REQUESTED** [review]

**Pull request merged**